src/stretchy.js

```diff
@@ -193,61 +192,52 @@ const svgSpan = function(group, options) {
         attributes.push(["preserveAspectRatio", "none"]);
 
         svgNode = new domTree.svgNode([path], attributes);
+        span = buildCommon.makeSpan([], [svgNode], options);
 
     } else {
-        let width;
+        let widthClass;
         let align;
+        const spans = [];
 
         const [paths, gWidth, vbHeight, alignOne] = katexImagesData[label];
         const numSvgChildren = paths.length;
-        const innerSVGs = [];
         height = vbHeight / 1000;
         minWidth = gWidth;
 
         for (let i = 0; i < numSvgChildren; i++) {
             path = new domTree.pathNode(paths[i]);
-            attributes = [];
+
+            attributes = [["width", "400em"], ["height", height + "em"]];
+            attributes.push(["viewBox", `0 0 ${viewBoxWidth} ${vbHeight}`]);
 
             if (numSvgChildren === 1) {
-                width = "400em";
                 align = alignOne;
             } else if (numSvgChildren === 2) {
-                // small overlap to prevent a 1 pixel gap.
-                if (i > 0) {
-                    attributes.push(["x", "50%"]);
-                }
-                width = ["50.1%", "50%"][i];
+                widthClass = ["halfarrow-left", "halfarrow-right"][i];
                 align = ["xMinYMin", "xMaxYMin"][i];
             } else {
                 // 3 inner SVGs, as in a brace
-                if (i > 0) {
-                    attributes.push(["x", [null, "25%", "74.9%"][i]]);
-                }
-                width = ["25.5%", "50%", "25.1%"][i];
+                widthClass = ["brace-left", "brace-center", "brace-right"][i];
                 align = ["xMinYMin", "xMidYMin", "xMaxYMin"][i];
```

Comment:
This works b/c the SVG for each part of the brace extends out the right, in both directions, and to the left if we were looking at each pieces from left to right.

Reply:
Exactly.